chromium_src/chrome/browser/component_updater/file_type_policies_component_installer.cc

```diff
@@ -0,0 +1,27 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#define RegisterFileTypePoliciesComponent RegisterFileTypePoliciesComponent_ChromiumImpl  // NOLINT
+#include "../../../../../chrome/browser/component_updater/file_type_policies_component_installer.cc"  // NOLINT
+#undef RegisterFileTypePoliciesComponent
+
+#include "chrome/browser/component_updater/component_updater_utils.h"
+#include "components/component_updater/component_updater_service.h"
+#include "extensions/common/constants.h"
```

Comment:
as mentioned https://github.com/brave/brave-core/pull/6763/files#r508049285 this is problematic because it should be in an buildflag guard for extensions